### ISSUE TITLE
Can not edit big seatmaps

### ISSUE THREAD

**Issue by GradSimon** [lansuite/lansuite#1203]
We are currently experience a problem adding our seating to the system, once we reach an unkown amount of elements in the drawing it wont let us into the editing site anymore. Also during the save of the inital creation the website does not responde anymore, altough it should do the saving, since we can view the entered data afterwards.

Our current settings are 48 vertical and 8 horizontal, horizontally aligned, a much smaller test seatmap works fine, since there arent that many elements there.
![grafik](https://github.com/user-attachments/assets/2b4b3f2a-56bc-4322-9ba9-ea4dc3fc3d55)

![grafik](https://github.com/user-attachments/assets/9dd1b145-71b0-448f-94da-66139826bc3c)

Any help or suggestions would be appreciated :)

**Comment by M4LuZ** [comment]
Hi @GradSimon ,

I cannot reproduce the issue, that size (and much bigger, this is 59x99, the max value) works fine for me.
![grafik](https://github.com/user-attachments/assets/24071e29-6194-4cf6-9844-1661112fa01a)

If smaller selections work fine then I would suggest to look for any potential PHP limits that may be hit.
memory_limit, max_execution_time, suhoshin values - if used.
But all of these should leave an error messgage in the server logs when hit.

So my guess would be that this is related to your server config.
...which would be good to be listed. 
Not that we ask for that in the issue template without reason ;)

**Comment by M4LuZ** [comment]
Right, after a bit of investigation I found an issue, not sure if it is yours:
PHP introduced a variable `max_input_vars` the maximum amount of elements in a request.
The default value is 1000.

The amount of elements the seating plan uses is columns/rows (depending on orientation) *100 + <the other direction>.
This higher elements will be cut off.

Increasing the allowed amount of elements solves the issue.
Note that this also requires a good amount of memory, so you may also need to increase `memory_limit` to a higher value (which is most probably the issue behind your not loading page).

We may add a bit of documentation or sense-checking later on, but this should solve your issue immediately
